docs/data/base/components/snackbar/TransitionComponentSnackbar.tsx

```diff
@@ -0,0 +1,198 @@
+import * as React from 'react';
```

Comment:
_Note_: I have copied the design for this demo from [Chakra UI](https://chakra-ui.com/docs/components/feedback/toast#usage)

Reply:
cc @danilo-leal maybe we want to re-adjust the styles so that we have our own instead of copied from another library :)

Reply:
Done some tinkering on the demos there 👌

Reply:
Thanks Danilo! Can you also take a look at the [third demo](https://deploy-preview-33227--material-ui.netlify.app/base/react-snackbar/#the-usesnackbar-hook) I added now for the `useSnackbar` hook? Perhaps you want to tweak it as well.